### ISSUE TITLE
Error building C# project: Some projects referenced in my solution file not found

### ISSUE THREAD

**Issue by rickyah** [travis-ci/travis-ci#4039]
Hi, I've got a .NET project that I'm trying to get built with the beta support of C#
This is the project with the branch I'm using to change the configuration:
https://github.com/rickyah/ini-parser/tree/update-travisci

My solution file references three projects. The problem I found is that when the travis machine builds the project warms me about two of the three projects cannot be found and will be ignored. One of them is the unit tests projects, as it does not get build and the run test step fails.

I've also built the project using mono in my dev machine (OSX) using the command line, and all the steps execute ok: nuget packages are restored, all three projects are built and all tests pass.

I've printed the directory tree as the first step before starting the build in an attempt to figure out what was happening and I can confirm that the two other projects does exist in the build folder, but I don't know why is failing.
Here's the latests build log:
https://travis-ci.org/rickyah/ini-parser/jobs/65732192

cc @joshua-anderson @akoeplinger @nterry

**Comment by akoeplinger** [comment]
The Travis machine runs Linux which has a case-sensitive file system by default. Your sln specifies `IniFileParser.Tests.csproj`, but the actual file on disk is called `INIFileParser.Tests.csproj`. Change the casing to match and it should start to work ;)

**Comment by rickyah** [comment]
Ok it a really weird strange because I've actually checked that out. Turns out my local copy does have the file on disk called `IniFileParser.Tests.csproj` However I've cloned my repo again in a different folder, and you are right: on a fresh clone the file appears `INIFileParser.Tests.csproj' 
This is SO weird, maybe a rename not tracked correctly? 

Either way that fixed the problem, but there are a bunch of test files that failed for the same reason. Gotta change to a case-sensisitve FS if I ever reinstall the OS from scratch. Thanks!
